modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/util/DLProcessorRegistryImpl.java

```diff
@@ -268,15 +269,17 @@ private FileVersion _getLatestFileVersion(
 		try {
 			return fileEntry.getLatestFileVersion(trusted);
 		}
-		catch (NoSuchFileEntryException noSuchFileEntryException) {
-			if (_log.isInfoEnabled()) {
-				_log.info(noSuchFileEntryException);
-			}
-
-			return null;
-		}
 		catch (Exception exception) {
-			_log.error(exception);
+			if (exception instanceof NoSuchFileEntryException ||
```

Comment:
@yuhai you can keep the original catch and append `NoSuchFileVersionException` to it. Please check "multi catch" topic on the internet.

On the other hand, do you mean that an LPS made the `getLatestFileVersion` throw `NoSuchFileVersionException ` in addition to `NoSuchFileEntryException`, but forgot to update the catch logic here? If so, please just state this in the commit message. No need to say "it will fix xxxTest".